scss/_utilities_hide.scss

```diff
@@ -21,19 +21,19 @@
     display: none !important;
 
     &--small {
-      @media (max-width: $breakpoint-small - 1) {
+      @media (width < $breakpoint-small) {
         display: none !important;
       }
     }
 
     &--medium {
-      @media (min-width: $breakpoint-small) and (max-width: $breakpoint-large - 1) {
+      @media ($breakpoint-small <= width < $breakpoint-large) {
         display: none !important;
       }
     }
 
     &--large {
-      @media (min-width: $breakpoint-large) {
+      @media ($breakpoint-large <= width) {
```

Comment:
There is some inconsistency throughout this PR with whether `width` is on the left or right side of the inequality that makes it a bit harder to review. I propose this standard, but I'm open to ideas:

1. Both bounds are known: `min < width < max`
2. Only on bound is known: `width < max` or `width > min` (width always on the left)

Reply:
Okay, that's a sound explanation - I can live with it. Thanks!